server/config/immutable_options.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+// ImmutableConfig is a readonly config.
+type ImmutableConfig struct {
+	maxRegionSize uint64
+	leaderLease   int64
```

Comment:
Why only leaderLease?

Reply:
`leaderLease` will be  remove and add `max-split-size`.